### PR TITLE
feat(string):added camelCase function

### PR DESCRIPTION
**Pull request by BethiPooja**
## Summary by CodeRabbit

- **New Features**
	- Introduced a new function to convert strings into camel case format.
	- Made the camel case function easily accessible throughout the application.
  
- **Tests**
	- Added comprehensive tests to ensure the functionality of the camel case conversion, covering various scenarios.

### CODE REVIEW

**Comment by BethiPooja** [comment]
I'm closing this PR because I haven't written a title and description.

File: src/utils/string/camelCase/camelCase.test.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { camelCase } from "./camelCase";
+
+function testCamelCase() {
```
Comment (bharat-dussa) [inline src/utils/string/camelCase/camelCase.test.ts:3]: This function is not required .

File: src/utils/string/camelCase/camelCase.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+export function camelCase(str: string): string {
+    let words = str.split(' ');
+    let  camelCaseStr = words[0].toLowerCase();
```
Comment (bharat-dussa) [inline src/utils/string/camelCase/camelCase.ts:3]: handle for length == 0

Comment (bharat-dussa) [reply src/utils/string/camelCase/camelCase.ts:3]: trim the input

**Review by bharat-dussa — CHANGES_REQUESTED** [review]

**Comment by bharat-dussa** [comment]
Please raise a new PR since you have deleted HEAD Repo

**Pull request closed without merging**